WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -408,11 +409,13 @@ fileprivate func < <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         if contentProvider == nil || contentProvider?.sourceAttributionStyle() != SourceAttributionStyle.none {
             resetActionButton(commentActionButton)
             resetActionButton(likeActionButton)
+            resetActionButton(saveForLaterButton)
             return
         }
 
         configureCommentActionButton()
         configureLikeActionButton()
+        configureSaveForLaterButton()
```

Comment:
We should only call this (or add a guard into the method) if the feature flag is enabled. I know currently the button won't be highlighted, as `isSavedForLater` will be false, but we may end up putting more things in that method later.